src/engine/eip7928.md

```diff
@@ -75,9 +75,11 @@ This method follows the same specification as [`engine_newPayloadV4`](./prague.m
 
 1. Client software **MUST** return `-38005: Unsupported fork` error if the `timestamp` of the payload does not fall within the time frame of the EIP-7928 activation.
 
-2. Client software **MUST** validate the `blockAccessList` field by executing the payload's transactions and verifying that the computed access list matches the provided one.
+2. Client software **MUST** return `-32602: Invalid params` error if the `blockAccessList` field is missing.
 
-3. If the `blockAccessList` field is missing, malformed, or doesn't match the computed access list, the call **MUST** return `{status: INVALID, latestValidHash: null, validationError: errorMessage | null}`.
+3. Client software **MUST** validate the `blockAccessList` field by executing the payload's transactions and verifying that the computed access list matches the provided one.
```

Comment:
Can we simplify and say: 
> Client software **MUST** validate the `blockAccessList` field by executing the payload's transactions and verifying that the computed access list matches the provided one. If it doesn’t match, the call **MUST** return `{status: INVALID, latestValidHash: null, validationError: errorMessage | null}`.

In this context “doesn’t match” should cover the “malformed” case as well, shouldn’t it?
Also, having validation check and the corresponding failure response in one statement is better for readability IMHO